junit-jupiter-api/src/main/java/org/junit/jupiter/api/condition/AbstractJreRangeCondition.java

```diff
@@ -70,9 +70,6 @@ protected final boolean isCurrentVersionWithinRange(JRE minJre, JRE maxJre, int
 		// Finally, we need to validate the effective minimum and maximum values.
 		Preconditions.condition((min != JRE.MINIMUM_VERSION || max != Integer.MAX_VALUE),
 			() -> "You must declare a non-default value for the minimum or maximum value in @" + this.annotationName);
-		Preconditions.condition(min >= JRE.MINIMUM_VERSION,
-			() -> String.format("@%s's minimum value [%d] must greater than or equal to %d", this.annotationName, min,
-				JRE.MINIMUM_VERSION));
 		Preconditions.condition(min <= max,
```

Comment:
I'm guessing that's logically impossible based on previous precondition checks higher up in that method -- yes?

Reply:
Never mind.

I originally didn't see the commit message which already answers that.